content/en/docs/ops/ambient/getting-started/test.sh

```diff
@@ -62,10 +62,9 @@ _verify_contains snip_verify_traffic_sleep_to_ingress "$snip_verify_traffic_slee
 _verify_contains snip_verify_traffic_sleep_to_productpage "$snip_verify_traffic_sleep_to_productpage_out"
 _verify_contains snip_verify_traffic_notsleep_to_productpage "$snip_verify_traffic_notsleep_to_productpage_out"
 
-snip_adding_your_application_to_the_ambient_mesh_1
+_verify_contains snip_adding_your_application_to_the_ambient_mesh_1 "$snip_adding_your_application_to_the_ambient_mesh_1_out"
```

Comment:
I'm not surprised it's failing now, since you're not calling snip_adding_your_application_to_the_ambient_mesh_2

Why didn't you change to the code I suggested in my previous comment? https://github.com/istio/istio.io/pull/14788#pullrequestreview-1964009799 
```suggestion
_verify_contains snip_adding_your_application_to_the_ambient_mesh_1 "$snip_adding_your_application_to_the_ambient_mesh_1_out"
snip_adding_your_application_to_the_ambient_mesh_2
```

Reply:
I'm not calling it because when I generate the snips there is no `snip_adding_your_application_to_the_ambient_mesh_2` and that is because as pointed out before there is no need to validate this section:
```
{{< text syntax=bash snip_id=none >}}
$ kubectl label namespace default istio.io/dataplane-mode=ambient
{{< /text >}}
```
as I'm not referencing any output on the document.

Are you saying that even though in this case I should remove the `snip_id=none ` let it generate the `snip_adding_your_application_to_the_ambient_mesh_2` and just not call any validation from the test.sh?

Reply:
I said that you don't need to validate the output of the snip, not that you don't need to call it. If you don't call the snip that adds the label, how could it possibly get labelled so that the following snips will work?

Just put it back to generate all the snips the way you had it previously and change the test to what I said yesterday.
```
_verify_same snip_adding_your_application_to_the_ambient_mesh_1 "$snip_adding_your_application_to_the_ambient_mesh_1_out"

# test traffic after ambient mode is enabled
snip_adding_your_application_to_the_ambient_mesh_2
_verify_like snip_adding_your_application_to_the_ambient_mesh_3 "$snip_adding_your_application_to_the_ambient_mesh_3_out"
_verify_same snip_adding_your_application_to_the_ambient_mesh_4 "$snip_adding_your_application_to_the_ambient_mesh_4_out"
_verify_same snip_adding_your_application_to_the_ambient_mesh_5 "$snip_adding_your_application_to_the_ambient_mesh_5_out"
_verify_same snip_adding_your_application_to_the_ambient_mesh_6 "$snip_adding_your_application_to_the_ambient_mesh_6_out"
```
Either that, or disable the test (rename it to `test-disable.sh`) and I'll fix it in a followup.

Reply:
`_verify_same` and `_verify_contains` will both work in these cases, but `_verify_same` is a better check since it really is checking that the output is "exactly the same as expected", which it is in this case. `_verify_contains` checks that the expected string is part of the output, which could include more text.

Reply:
https://github.com/istio/istio.io/blob/master/tests/README.md#verify-functions

Reply:
understood, I think I tested with that and succeed locally but failed here. I will send one like this and we'll see, thanks.